librosa/display.py

```diff
@@ -818,17 +852,41 @@ def __decorate_axis(axis, ax_type):
         axis.set_label_text('BPM')
 
     elif ax_type == 'time':
-        axis.set_major_formatter(TimeFormatter(lag=False))
+        axis.set_major_formatter(TimeFormatter(unit=None, lag=False))
```

Comment:
This isn't actually necessary; since it's calling with explicit keywords, order does not matter.  I'd say it's fine as is.

Reply:
yes, it does not change the behavior of the program, but I figured it might be good to "set the example" right in the code base. but if you say that that's an overcorrection, let's forget about it